models/Asset/Image/Thumbnail.php

```diff
@@ -308,19 +308,10 @@ public function getHtml($options = [])
                 $sourceHtml = $this->getSourceTagHtml($thumbConfig, $mediaQuery, $image, $options);
                 if (!empty($sourceHtml)) {
                     if ($isAutoFormat) {
-                        $autoFormats = \Pimcore::getContainer()->getParameter('pimcore.config')['assets']['image']['thumbnails']['auto_formats'];
-                        foreach ($autoFormats as $autoFormat => $autoFormatConfig) {
-                            if (self::supportsFormat($autoFormat) && $autoFormatConfig['enabled']) {
-                                $thumbConfigAutoFormat = clone $thumbConfig;
-                                $thumbConfigAutoFormat->setFormat($autoFormat);
-                                if (!empty($autoFormatConfig['quality'])) {
-                                    $thumbConfigAutoFormat->setQuality($autoFormatConfig['quality']);
-                                }
-
-                                $sourceWebP = $this->getSourceTagHtml($thumbConfigAutoFormat, $mediaQuery, $image, $options);
-                                if (!empty($sourceWebP)) {
-                                    $html .= "\t" . $sourceWebP . "\n";
-                                }
+                        foreach ($thumbConfig->getAutoFormatThumbnailConfigs() as $autoFormatConfig) {
+                            $sourceWebP = $this->getSourceTagHtml($autoFormatConfig, $mediaQuery, $image, $options);
```

Comment:
Maybe we should use a more generic variable name here instead of `$sourceWebP`.

Reply:
Changed it to `$autoFormatThumbnailHtml`